include/dp_conf_opts.h

```diff
@@ -11,6 +11,7 @@
 enum dp_conf_nic_type {
 	DP_CONF_NIC_TYPE_HW,
 	DP_CONF_NIC_TYPE_TAP,
+	DP_CONF_NIC_TYPE_BLUEFIELD2,
```

Comment:
Maybe it's not a good idea due to a breaking change, but HW/TAP/Bluefield seems strange, as previously this meant a switch between "real" and "virtual" cards, it now is more of a "type". Then I would suggest changing "hw" to "nvidia" or "mellanox" or "sriov".

The breaking change should only be visible in the docker container which is part of this repository, so it should not break much and as for OSC, there is the whole ironcore switch so breaks are expected anyway.

Reply:
Thanks for the suggestions. I think breaking change is not so problematic in this case.
I like Mellanox better. So we can simply change HW with Mellanox as below:

> enum dp_conf_nic_type {
	DP_CONF_NIC_TYPE_MELLANOX,
	DP_CONF_NIC_TYPE_TAP,
	DP_CONF_NIC_TYPE_BLUEFIELD2,
}